contents/handbook/why-does-posthog-exist.md

```diff
@@ -6,29 +6,63 @@ showTitle: true
 
 ## Our mission
 
-Equip every developer to build successful products
+Equip every developer to build successful products.
+
+## A few words
+
+As you know, the initial products of PostHog are a bunch of different SaaS tools, which work together better than everyone else’s. However, what most people may not know, is that our long term plan is to automate how entire businesses operate built around the unit they operate on - their customer data.
+
+Critical to making this happen is to deliver each tool without compromises. The way to achieve this is very simple - shipping every product that teams use, from day one when there are just two technical cofounders. This is because if we get in with these developers, we’re upstream of every need they’ll have. The end result is that we can offer all the products in one place, with no need to charge as much as everyone else for each one, and most importantly with perfect quality customer data. An actual perfect 360 customer view.
+
+Even so, it’s right to question whether this actually does any good for the world. Are we really in need of better software? And beyond those in the most competitive industries, will perfect customer data actually make a difference to an everyday person’s ability to build a successful business?
+
+No and not really. However, that misses the point, unless you understand the secret master plan alluded to above. The strategy of PostHog is to get early stage software teams to use every single customer related product inside our platform, where we can easily generate a perfect customer data record. This means we can learn to configure and automate how all of these tools work. We can simplify the experience of running a business.
+
+Then with each successive product, we can reach a wider and wider market and further simplify the experience of running a business until anyone can do it.
```

Comment:
But how does _this_ do any good for the world? Is it because successful businesses make the world a better place? Because business owners are happier people? Because more businesses means more options for consumers? It could be any of those..... but the way it's written it just feels like this leaves a gap.

Reply:
i think all of the above - like the world is more productive so living standards are higher, it's more accessible to create a company, the world is more inspiring and less "set in stone", so will leave intentionally vague!